docs/en/user_guides/train_test.md

```diff
@@ -0,0 +1,350 @@
+
+### Datasets for image translation models
+
+For translation models, now we offer two settings for datasets called paired image dataset and unpaired image dataset.
+
+For paired image dataset, every image is formed by concatenating two corresponding images from two domains along the width dimension. You are supposed to make two folders "train" and "test" filled with images of this format for training and testing. Folder structure is presented below.
+
+```
+./data/dataset_name/
+├── test
+│   └── XXX.jpg
+└── train
+    └── XXX.jpg
+
+```
+
+For unpaired image dataset, you are supposed to make two folders "trainA" and "testA" filled with images from domain A and two folders "trainB" and "testB" filled with images from domain B. Folder structure is presented below.
+
+```
+./data/dataset_name/
+├── testA
+│   └── XXX.jpg
+├── testB
+│   └── XXX.jpg
+├── trainA
+│   └── XXX.jpg
+└── trainB
+    └── XXX.jpg
+
+```
+
+Please read the section `Datasets for unconditional models` and also use the symlink (soft link) to build up the dataset.
+
+Here, we provide download links of datasets used in [Pix2Pix](http://efrosgans.eecs.berkeley.edu/pix2pix/datasets/) and [CycleGAN](https://people.eecs.berkeley.edu/~taesung_park/CycleGAN/datasets/).
+
+## Train existing models
+
+Currently, we have tested all of the model on distributed training. Thus, we highly recommend to adopt distributed training with our scripts. The basic usage is as follows:
+
+```shell
+sh tools/dist_train.sh ${CONFIG_FILE} ${GPUS_NUMBER} \
+    --work-dir ./work_dirs/experiments/experiments_name \
+    [optional arguments]
+```
+
+If you are using slurm system, the following commands can help you start training"
+
+```shell
+sh tools/slurm_train.sh ${PARTITION} ${JOB_NAME} ${CONFIG} ${WORK_DIR} \
+    [optional arguments]
+```
+
+There two scripts wrap [tools/train.py](https://github.com/open-mmlab/mmgeneration/tree/master/tools/train.py) with distributed training entrypoint. The `optional arguments` are defined in [tools/train.py](https://github.com/open-mmlab/mmgeneration/tree/master/tools/train.py). Users can also set `random-seed` and `resume-from` with these arguments.
+
+Note that the name of `work_dirs` has already been put into our `.gitignore` file. Users can put any files here without concern about changing git related files. Here is an example command that we use to train our `1024x1024 StyleGAN2 ` model.
+
+```shell
+sh tools/slurm_train.sh openmmlab-platform stylegan2-1024 \
+    configs/styleganv2/stylegan2_c2_ffhq_1024_b4x8.py \
+    work_dirs/experiments/stylegan2_c2_ffhq_1024_b4x8
+```
+
+During training, log files and checkpoints will be saved to the working directory. At the beginning of our development, we evaluate our model after the training finishes. However, the evaluation hook has been already supported to evaluate our models in the training procedure. More details can be found in our tutorial for running time configuration.
```

Comment:
"At the beginning of our development, we evaluate our model after the training finishes."
We support evaluating model during the training process in 0.x version as well. Maybe we can delete this sentense.